pkg/security/admission/admission_test.go

```diff
@@ -68,9 +68,11 @@ func TestAdmit(t *testing.T) {
 			Type: kapi.SupplementalGroupsStrategyMustRunAs,
 		},
 		Groups: []string{"system:serviceaccounts"},
+		// give this scc priority since it is what we want to validate with
+		SortPriority: 1,
```

Comment:
actually, I don't need this priority at all.  The SCC below it is more restrictive (a MustRunAs uid strategy rather than a MustRunAsRange) that is in `scc-sa`.  Removing the priority they would have equal priorities.  The scores should ensure that `scc-sa-exact` is tried before `scc-sa` but it should be failing to validate.  We check the matched name in the test run which will ensure we're matching against what we expect....removing.